integrationtests/gitcloner/clone_test.go

```diff
@@ -504,6 +566,27 @@ func createAndAddKeys() (string, error) {
 	return privateKey, nil
 }
 
+func getKnownHostEntry(container testcontainers.Container) (string, error) {
+	mappedPort, err := container.MappedPort(context.Background(), gogsSSHPort)
+	if err != nil {
+		return "", err
+	}
+	port := mappedPort.Port()
+
+	publicHostKey, err := container.CopyFileFromContainer(context.Background(), "/data/ssh/ssh_host_ecdsa_key.pub")
+	if err != nil {
+		return "", err
+	}
+	publicHostKeyBytes, err := io.ReadAll(publicHostKey)
+	if err != nil {
+		return "", err
+	}
+	algorithm := strings.Split(string(publicHostKeyBytes), " ")[0]
+	hostKey := strings.Split(string(publicHostKeyBytes), " ")[1]
```

Comment:
nit: why split the public host key twice?
```suggestion
        fields := strings.Split(string(publicHostKeyBytes), " ")
	algorithm := fields[0]
	hostKey := fields[1]
```